### PR TITLE
add logistic and mixed-effects

### PR DESCRIPTION
**Pull request by boyinggong**

### CODE REVIEW

File: slides/05_results.md [hunk 1]
```diff
@@ -2,6 +2,34 @@
 
 ## Behaviral Analysis
```
Comment (BenjaminHsieh) [inline slides/05_results.md]: spelling behavioral

File: slides/05_results.md [hunk 2]
```diff
@@ -2,6 +2,34 @@
...
+
+- Model diagnosis
+\begin{itemize}
+\item Accuracy of Logistic models on the training set:\\
+median = 89.78\% min = 80.97\% max = 99.21\%
+\item Accuracy using 10-fold cross-validation: \\
+median = 89.86\% min = 79.92\% max = 98.45\%
+\end{itemize}
+
+## Behaviral Analysis (Continued)
```
Comment (BenjaminHsieh) [inline slides/05_results.md]: spelling

File: slides/05_results.md [hunk 3]
```diff
@@ -2,6 +2,34 @@
 
 ## Behaviral Analysis
 
+- Fit Logistic regression model on the behavioral data
+\begin{equation}
+\textrm{log}\frac{p(X)}{1-p(X)} = \beta_0 + \beta_{loss} *X_{loss} + \beta_{gain} * X_{gain}
+\end{equation}
+where the response are categorical variable indicating acceptance or rejection of a gamble.
+
+- Calculate the behavoral loss aversion
```
Comment (BenjaminHsieh) [inline slides/05_results.md]: spelling

**Comment by BenjaminHsieh** [comment]
Also will the slides in file 03_methods be updated? If not we can delete them. 

Also @brianqiu do we have any figures/results from preprocessing, let us know!

**Comment by boyinggong** [comment]
the model part will no loner be updating. We put the models and results together.

**Comment by briankleinqiu** [comment]
Hey sorry guys didn't have access to computer tell now. Will add by midnight

**Comment by briankleinqiu** [comment]
Approximately is has typoes in first slide :p doesn't really matter tho. Looks good, pull everything and merge to master now?

**Comment by BenjaminHsieh** [comment]
sure

**Pull request merged**